frameworks/elastic/tests/test_tls.py

```diff
@@ -30,46 +33,69 @@ def service_account(configure_security):
 
 @pytest.fixture(scope='module')
 def elastic_service_tls(service_account):
-    sdk_install.install(
-        config.PACKAGE_NAME,
-        service_name=config.SERVICE_NAME,
-        expected_running_tasks=config.DEFAULT_TASK_COUNT,
-        additional_options={
-            "service": {
-                "service_account_secret": service_account["name"],
-                "service_account": service_account["secret"],
-                "security": {
-                    "transport_encryption": {
-                        "enabled": True
+    try:
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            service_name=config.SERVICE_NAME,
+            expected_running_tasks=config.DEFAULT_TASK_COUNT,
+            additional_options={
+                "service": {
+                    "service_account_secret": service_account["name"],
+                    "service_account": service_account["secret"],
+                    "security": {
+                        "transport_encryption": {
+                            "enabled": True
+                        }
                     }
+                },
+                "elasticsearch": {
+                    "xpack_enabled": True,
                 }
-            },
-            "elasticsearch": {
-                "xpack_enabled": True,
             }
-        }
-    )
+        )
 
-    sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
+        sdk_plan.wait_for_completed_deployment(config.SERVICE_NAME)
 
-    # Wait for service health check to pass
-    shakedown.service_healthy(config.SERVICE_NAME)
+        # Wait for service health check to pass
+        shakedown.service_healthy(config.SERVICE_NAME)
```

Comment:
Is this valid? As far as I know, the scheduler returns healthy long before the deployment has completed.

Reply:
I think this would only be needed/desired for services that define health checks. Since Elastic doesn't and  `sdk_install.install` by default waits for the deployment to finish we can remove it.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2349/commits/02ad4ea8eb1318f79cff4dd6015499ea0895739a